pkgs/development/libraries/jellyfin-ffmpeg/default.nix

```diff
@@ -1,21 +1,21 @@
-{ ffmpeg_5-full
-, nv-codec-headers-11
+{ ffmpeg_6-full
+, nv-codec-headers-12
 , chromaprint
 , fetchFromGitHub
 , lib
 }:
 
-(ffmpeg_5-full.override {
-  nv-codec-headers = nv-codec-headers-11;
+(ffmpeg_6-full.override {
+  nv-codec-headers-11 = nv-codec-headers-12;
```

Comment:
This is confusing. Is this intended?

Reply:
This is a quirk of the ffmpeg derivation, it takes both unversioned `nv-codec-headers` (currently v9) and `nv-codec-headers-11`):

https://github.com/NixOS/nixpkgs/blob/ac5000d72beb3cdcae1c8cf81b4fd71f696ce402/pkgs/development/libraries/ffmpeg/generic.nix#L232-L233

and then depending on the version of FFmpeg source it chooses the version of headers used:

https://github.com/NixOS/nixpkgs/blob/ac5000d72beb3cdcae1c8cf81b4fd71f696ce402/pkgs/development/libraries/ffmpeg/generic.nix#L555

As I am not sure why it is done this way (rather than a traditional override or similar), and this is the first usage of v12, I decided to not mess with it.

Reply:
I see. Changing that would indeed be out of scope.